bin/public/horizontal-split.html

```diff
@@ -1,24 +0,0 @@
-<!DOCTYPE html>
```

Comment:
I added a horizontal look as well, Please head to localhost/doublerun and for one view localhost/run.
I was wondering if your router had something similar to angular ngview?